### PR TITLE
Fixes #12352 - fix logging when orchestration fails

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/lib/actions/foreman_chef/client/destroy.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def plan(fqdn, proxy)
             end
           end
         rescue => e
-          Foreman::Logging.exception('Unable to communicate with Chef proxy', e)
+          ::Foreman::Logging.exception('Unable to communicate with Chef proxy', e)
```
Comment (ehelms) [inline app/lib/actions/foreman_chef/client/destroy.rb]: You might consider changing this over to use the plugins logger as well, e.g.:

```
::Foreman::Logging.logger('foreman_chef').exception()
```

**Comment by mmoll** [comment]
what's the status here?

**Comment by ares** [comment]
fixed

**Comment by ares** [comment]
merging

**Comment by mmoll** [comment]
As this just popped up on IRC again, it would be very nice to have a released version of this plugin for at least 1.10 with this fix in.

**Comment by ares** [comment]
@mmoll ok, wil try to do the release soon

**Pull request merged**